filer/private/sass/components/_filter-files.scss

```diff
@@ -4,22 +4,27 @@
 .filter-files-container {
     @include clearfix;
     position: relative;
-    float: left;
+    float: right;
+    width: calc(60% - 420px);
+    min-width: 170px;
     margin: 0 10px 0 0;
     padding: 0;
     background: none;
     box-shadow: none;
+    transition: width $speed-base;
+    z-index: 100;
```

Comment:
we should start using it (maybe in the next PR) 
this would help tremendously as the app grows complex